pkg/apis/crd/v1beta1/types.go

```diff
@@ -1054,6 +1054,9 @@ const (
 	// ActionForwardedOutOfOverlay indicates that the packet has been forwarded out of the network
 	// managed by Antrea. This indicates that the Traceflow request can be considered complete.
 	ActionForwardedOutOfOverlay TraceflowAction = "ForwardedOutOfOverlay"
+	// ActionForwardedOutOfScope indicates that the packet has been forwarded out of the network
+	// managed by Antrea, but it's not a full end-to-end observation.
+	ActionForwardedOutOfScope   TraceflowAction = "ForwardedOutOfScope"
```

Comment:
Am I understanding this correctly that `ForwardedOutOfScope` is supposed to be the equivalent to `ForwardedOutOfOverlay`, but for noEncap (as maybe `ForwardedOutOfOverlay` is specific to encap mode)?

If yes, should we have a unique observation name for both cases? We could deprecate `ForwardedOutOfOverlay`.
Personally I would prefer `ForwardedOutOfNetwork` to `ForwardedOutOfScope`.

Reply:
> but it's not a full end-to-end observation

how is it different from `ForwardedOutOfOverlay` in this regard? For Pod-to-External traffic, `ForwardedOutOfOverlay` is reported when the traffic is output on the gw interface. It is the final observation and you could say we don't have a full end-to-end observation as we don't have any information about the packet being delivered to its (external) destination.

Reply:
We can perhaps unify the 2 enums in a future API version

Reply:
Added TODO here.